src/modules/launcher/Plugins/Microsoft.Plugin.WindowWalker/Components/Window.cs

```diff
@@ -71,7 +71,7 @@ public IntPtr Hwnd
         public uint ProcessID { get; set; }
 
         /// <summary>
-        /// Gets returns the name of the process
+        /// Gets the name of the process
```

Comment:
Is this build breaking or a suggestion. If it's not build breaking it makes more sense if I fix it in my PR. I reworked the class and we will get conflicts.

Reply:
neither, I just fixed a wrong comment :)